src/dbus_api/api.rs

```diff
@@ -82,18 +82,18 @@ fn object_path_to_pair(dbus_context: &DbusContext,
 }
 
 /// Get name for pool from object path
-fn object_path_to_pool_uuid(dbus_context: &DbusContext,
+fn pool_object_path_to_pair(dbus_context: &DbusContext,
                             path: &dbus::Path)
-                            -> Result<Uuid, (MessageItem, MessageItem)> {
-    let uuid = match dbus_context.pools.borrow().get(path) {
+                            -> Result<(dbus::Path<'static>, Uuid), (MessageItem, MessageItem)> {
+    let pair = match dbus_context.pools.borrow().get(path) {
```

Comment:
This is just for uniformity? Maybe if they were sharing a function, but pool already has its own function, so adding an unuseful return parameter may cause confusion. I vote to drop this, unless there are other reasons...